cub/test/ptx-json/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+if ("${CMAKE_CXX_STANDARD}" STREQUAL "" OR "${CMAKE_CXX_STANDARD}" LESS "20")
```

Comment:
question: I think CUB presets do not define `CMAKE_CXX_STANDARD` but rather set dialect as property of specific targets. This means that in CI, we'll always skip this file. Is this intended?

Reply:
I think you're right, I'm going to figure out the correct invocations of the CUB dialect related macros.